src/datumaro/plugins/validators.py

```diff
@@ -547,11 +547,31 @@ def compute_statistics(self, dataset):
 
         stats, filtered_anns = self._compute_common_statistics(dataset)
 
+        label_cat = dataset.categories()[AnnotationType.label]
+        label_groups = label_cat.label_groups
+
+        label_name_to_group = {}
+        for label_group in label_groups:
+            for idx, label_name in enumerate(label_group.labels):
+                if label_group.group_type == 0:
```

Comment:
For better interpretability,
```suggestion
                if label_group.group_type == GroupType.EXCLUSIVE:
```

Reply:
Fixed. Thanks.